src/main/webapp/app/shared/user-settings/user-settings-prototype/user-settings-prototype.component.html

```diff
@@ -0,0 +1,42 @@
+<!--
+       Exemplary HTML template for UserSettings
+       Every settings page will be somewhat unique, especially its optionCore (different properties and methods)
+       This nested behavior could not be realized via polymorphism due to different types and strong bidirectional dependencies
+       Therefore just copy this template for new settings and modify it. The minimal adjustment needed for the template to work is to implement the
+       option core with its properties and functionalities under <! --OptionCore-- >
+       Look at notification-settings.component.html for a good example
+
+<jhi-alert></jhi-alert>
+<div class="d-flex justify-content-between">
+    <h1>
+        <! --UserSettingsCategory-- >
+        <! -- please look at the README.md file to understand how the translation works -- >
+        {{ 'artemisApp.userSettings.categories.' + userSettingsCategory | artemisTranslate }}
+    </h1>
+    <button type="button" class="btn btn-primary" *ngIf="optionsChanged" id="apply-changes-button" (click)="saveOptions()">
+        <fa-icon [icon]="'save'"></fa-icon>
+        {{ 'artemisApp.userSettings.saveChanges' | artemisTranslate }}
+    </button>
+</div>
+<div class="list-group d-block" *ngIf="userSettings">
+    <div *ngFor="let optionGroup of userSettings.groups">
+        <div *jhiHasAnyAuthority="[optionGroup.restrictionLevel]" class="list-group-item">
+            <h3>
+                <! --GroupName-- >
+                {{ 'artemisApp.userSettings.optionGroupNames.' + optionGroup.name | artemisTranslate }}
+            </h3>
+            <div *ngFor="let option of optionGroup.options">
+                <dt>
+                    <! --OptionName-- >
+                    {{ 'artemisApp.userSettings.optionNames.' + option.name | artemisTranslate }}
+                </dt>
+                <span>
+                    <! --OptionDescription-- >
+                    {{ 'artemisApp.userSettings.optionDescriptions.' + option.description | artemisTranslate }}
+                </span>
+                <! --Put Settings specific core properties & functions here-- >
+            </div>
+        </div>
+    </div>
```

Comment:
I'd argue that this is redundant and also not common to do. With the one instantiation of a user-setting component (`notification-settings.component.ts`) and it's according template (`notification-settings.component.html`) you already provide an example that others can use as orientation. With this "dummy"-component you increase the effort to maintain the code in this folder, as this has to be kept in sync with the `notification-settings.component.html`.

Reply:
Very good points. Yes, I removed the file and refactored the component into a directive thank you very much 🎉